schemas/2015-06-01/Microsoft.KeyVault.json

```diff
@@ -0,0 +1,209 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.KeyVault",
+    "description": "Microsoft KeyVault Resource Types",
+    "definitions": {
+        "keyPermission": {
+            "anyOf": [
```

Comment:
Thank you for the review!

This is an area where I could use some guidance, honestly.  So, unfortunately, yes - you can.  But this seems to conflict with recommendations.  For example, the following is accepted (and stored verbatim) by the API:

```json
"keys": [
  "list",
  "all",
  "all",
  "all",
  "list"
],
"secrets": [
  "all",
  "list",
  "list"
]
```

While that is accepted by the KeyVault provider, the PowerShell cmdlets explicitly forbid mixing of 'all' and other levels of permissions:
```
PS C:\Users\Robert> Set-AzureKeyVaultAccessPolicy <# ... #> -PermissionsToKeys all, list
Set-AzureKeyVaultAccessPolicy : Invalid permissions to keys. The list contains "all" along
with other permissions. Please remove "all" from the list or include only "all" in the list.

PS C:\Users\Robert> Set-AzureKeyVaultAccessPolicy <# ... #> -PermissionsToSecrets all, list
Set-AzureKeyVaultAccessPolicy : Invalid permissions to keys. The list contains "all" along
with other permissions. Please remove "all" from the list or include only "all" in the list.
```

So I can either go the way PowerShell did, and try to prevent setting 'all' along with other permissions, or I can allow mixing 'all' with other access levels.  My initial thought was to try to provide a consistent development experience, so I tried to emulate what PowerShell would accept, but this does not represent in truth everything the provider allows.  Do you have a preference here?